docs/root/start/start.rst

```diff
@@ -0,0 +1,154 @@
+.. _start:
+
+Getting Started
+===============
+
+This section gets you started with a very simple configuration, and provides some example configurations.
+
+Envoy does not currently provide separate pre-built binaries, but does provide Docker images. This is
+the fastest way to get started using Envoy. Should you wish to use Envoy outside of a
+Docker container, you will need to :ref:`build it <building>`.
+
+These examples use the :ref:`v2 Envoy API <envoy_api_reference>`, but use only the static configuration
+feature of the API, which is most useful for simple requirements. For more complex requirements
+:ref:`Dynamic Configuration <arch_overview_dynamic_config>` is supported.
+
+Quick Start to Run Simple Example
```

Comment:
Nit: this file will grow pretty quickly. It might make more sense to have each section in its own file.

Reply:
Sure, could we do that when it expands?

Reply:
It's up to you. If you just want to land this PR to get going and plan on following up with two other PRs for splitting up the content and expanding the "Simple Configuration" section, that's fine, I agree with the idea of breaking this up, since the "Simple Configuration" is distinct to the later bits on sandboxes. If you'd like to do that and think this PR is ready for final review, can give this another pass.

Reply:
@htuch Yes, I would like to get this one in quickly, as it touches a lot of files, and involves a top-level reorg. Subsequent work will be only in a couple of files and thus easier to maintain a PR. I think it's ready for final review.